SubmittingResults.md

```diff
@@ -2,8 +2,21 @@
-| Dhrystone             | Enclustra Mars PM3 with Mars ZX2 (Zynq Z-7010 processor) | 3077651.5 Dhrystones/s | 10     | [Run details](RunningBenchmarks.md#dhrystone-21) |
-| Whetstone             | Enclustra Mars PM3 with Mars ZX2 (Zynq Z-7010 processor) | 1462.08 Whetstones/s   | 10     | [Run details](RunningBenchmarks.md#whetstone)|
+| Benchmark/Application | Platform/SoC                                             | Result w/ unit (st. dev)        | # Runs | Notes               |
+|-----------------------|----------------------------------------------------------|---------------------------------|--------|---------------------|
+| CoreMark              | Enclustra Mars PM3 with Mars ZX2 (Zynq Z-7010 processor) | 4961.007 iterations/s           | 10     | [Run details](RunningBenchmarks.md#coremark) |
+| Dhrystone             | Enclustra Mars PM3 with Mars ZX2 (Zynq Z-7010 processor) | 3077651.5 Dhrystones/s          | 10     | [Run details](RunningBenchmarks.md#dhrystone-21) |
+| Whetstone             | Enclustra Mars PM3 with Mars ZX2 (Zynq Z-7010 processor) | 1462.08 Whetstones/s            | 10     | [Run details](RunningBenchmarks.md#whetstone)|
+|-----------------------|----------------------------------------------------------|---------------------------------|--------|---------------------|
+| CoreMark              | Serenity 16: COM Express with CentOS 7 (GCC v4.8.5)      | 7909.05 (42.45) iterations/s    | 10     | [Run details](RunningBenchmarks.md#coremark) |
+| Dhrystone (registers) | Serenity 16: COM Express with CentOS 7 (GCC v4.8.5)      | 5653594 (157858) Dhrystones/s   | 10     | [Run details](RunningBenchmarks.md#dhrystone-21) |
+| Dhrystone (no regs)   | Serenity 16: COM Express with CentOS 7 (GCC v4.8.5)      | 5664488 (163398) Dhrystones/s   | 10     | [Run details](RunningBenchmarks.md#dhrystone-21) |
+| Whetstone             | Serenity 16: COM Express with CentOS 7 (GCC v4.8.5)      | 12857.1 (752.9) Whetstones/s    | 10     | [Run details](RunningBenchmarks.md#whetstone)|
+|-----------------------|----------------------------------------------------------|---------------------------------|--------|---------------------|
+| CoreMark              | Serenity 16: COM Express with CentOS 7 (GCC v6.2.1)      |                                 | 10     | [Run details](RunningBenchmarks.md#coremark) |
+| Dhrystone (registers) | Serenity 16: COM Express with CentOS 7 (GCC v6.2.1)      |                                 | 10     | [Run details](RunningBenchmarks.md#dhrystone-21) |
+| Dhrystone (no regs)   | Serenity 16: COM Express with CentOS 7 (GCC v6.2.1)      |                                 | 10     | [Run details](RunningBenchmarks.md#dhrystone-21) |
+| Whetstone             | Serenity 16: COM Express with CentOS 7 (GCC v6.2.1)      |                                 | 10     | [Run details](RunningBenchmarks.md#whetstone)|
```

Comment:
If you're planning on benchmarking the Serenity with GCC 6.2.1, do you want me to hold off on merging this pull request until these numbers are in, or do you want to just pull these in on another pull request?

Reply:
Whatever works for you! I don't think that this data needs to be on an official repository urgently, so maybe it would be cleaner to wait until all of the data is in.